str/trtools/merge_str_runner.py

```diff
@@ -95,9 +95,11 @@ def main(
         if len(cpg_ids) != len(set(cpg_ids)):
             raise ValueError('Duplicate CPG IDs detected in sample list')
 
+        # TODO Convert to use mergeSTR --vcfs-list when that option is available
         trtools_job.command(
             f"""
-        mergeSTR --vcfs {",".join(batch_vcfs)} --out {trtools_job.vcf_output} --vcftype eh
+        cd ${{BATCH_TMPDIR}}/inputs
+        mergeSTR --vcfs `(echo {";echo ,".join(batch_vcfs)}) | sed 's|'${{BATCH_TMPDIR}}'/inputs/*||' | tr -d '\\n'` --out {trtools_job.vcf_output} --vcftype eh
```

Comment:
Can this be broken into multiple lines, and potentially set to a variable before hand - this improves the readability of the command line:

```suggestion
        VCFS=$(`$(echo {";echo ,".join(batch_vcfs)}) | sed 's|'${{BATCH_TMPDIR}}'/inputs/*||' | tr -d '\\n'`)
        mergeSTR \
            --vcfs $VCFS \
            --out {trtools_job.vcf_output} \
            --vcftype eh
```

Reply:
We could (modulo not adding erroneously nested `$(` and backticks, and modulo potential concerns about the maximum size of `VCFS`), but the truth is this is a temporary hack that will go away when we have a TRTools image that includes the `--vcfs-list` option — whichever way that happens.

So I made the minimal change that will evaporate again soon when we change it to `mergeSTR --vcfs-list {batch_vcfs_listfile}` instead, which will be more readable than **main**'s existing `{",".join(batch_vcfs)}` interpolation :shrug:

IMHO the thing that makes this unreadable is the intricacy of the two instances of `echo` due to the crazy comma-formatting needed…